build-gluster-org/jobs/user-update.yml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: update-admin-list
+    node: 
+    description: This job updates the jenkins job's admin-list with the members of gluster-maintainers 
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        url: git@github.com:gluster/build-jobs.git
+        credentials-id: github-credentials
+        branches:
+        - remotes/origin/master
+        basedir: "build-jobs"
+
+    builders:
+    - shell: |
+        cat << EOF > requirements.txt
+        gitdb==4.0.5
+        GitPython==3.1.9
+        EOF
+        pip install -r requirements.txt
+    
+    - python: !include-raw: ../scripts/user-update.py  
+
+    parameters:
+    - string:
+        description: 'Enter the job names (space seperated)' 
+        name: JOB_DETAILS 
```

Comment:
What this parameter is for?

Reply:
When I took a look at the jenkins jobs not all the jobs were having the admin-list field specified, so I made it explicit for the user to select which are those jobs which needs to be updated. May be we can discuss which are jobs are gonna need the admin-list to be updated and made change accordingly. If all the jobs need this field to be updated then we might have to make changes for that as well.

Reply:
I think it would be better if the script was smart enough to do it, since the less questions we ask to humans, the less risk we have the said human get it wrong. But I can see how autodetection could be hard (since indeed, not all admin-list need to be updated). 

What do people think about:
- check which files have a admin-list entry
- check if the name contain the word "regression" ?

Reply:
Yes it was one of ideas 
I can implement it